14/umbraco-cms/tutorials/creating-a-basic-website/setting-the-navigation-menu.md

```diff
@@ -12,59 +14,59 @@ To create dynamic navigation links from the published content nodes, follow thes
 1. Go to **Settings**.
 2. Expand the **Templates** folder from the **Templating** section.
 3. Open the **Master** template.
-4. Locate the `<!-- Navigation -->` tag (around line 19).
+4. Locate the `<!-- Navigation -->` tag (around line 20).
 5. Right below it, place the cursor on an empty line.
 6. Select **Query builder...** in the top-right side of the editor.
-7. Make sure it is set to say *"I want all content from my website"*.
+7. Make sure it is set to say "I want **all content** from **my website**".
 8. Click **Submit**.
 
-You now have the following snippet in your **Master** Template:
-
-```csharp
-@{
-	var selection = Umbraco.ContentAtRoot().FirstOrDefault()
-    .Children()
-    .Where(x => x.IsVisible());
-}
-<ul>
-	@foreach (var item in selection)
-	{
-		<li>
-			<a href="@item.Url()">@item.Name()</a>
-		</li>
+9. You now have the following snippet in your **Master** Template:
+
+	```csharp
+	@{
+		var selection = Umbraco.ContentAtRoot().FirstOrDefault()
+		.Children()
+		.Where(x => x.IsVisible());
+	}
+	<ul>
+		@foreach (var item in selection)
+		{
+			<li>
+				<a href="@item.Url()">@item.Name()</a>
+			</li>
+		}
+	</ul>
+	```
+
+10. This snippet needs to be merged with the navigation above it.
+
+11. The `<ul>` tag needs to be wrapped inside the `<div class="container">` and `<nav>` tags, and the classes need to be added to the correct tags as well.
+
+12. The final result will look like this:
```

Comment:
Are these steps? They should be written as steps in that case, starting with the action word.

Reply:
They were initially not steps but I added them as steps cz it looked funny with some statements between lists. Will start the steps with an action word 💪